### PR TITLE
Iv thea benchmarks

### PR DESCRIPTION
**Pull request by 35359595**
## Describe your changes

Thea pallet benchmarks implementation and weights integration for extrinsics;

## Checklist before requesting a review
- [x] I have performed a self-review of my code.
- [x] If it is a core feature, I have added thorough tests.
- [x] I removed all Clippy and Formatting Warnings. 
- [x] I added required Copyrights.

### CODE REVIEW

**Review by Gauthamastro — APPROVED** [review]

**Review by Gauthamastro — APPROVED** [review]

File: pallets/liquidity/src/weights.rs [hunk 1]
```diff
@@ -1,93 +1,106 @@
...
-// the Free Software Foundation, either version 3 of the License, or
-// (at your option) any later version.
-//
-// This program is distributed in the hope that it will be useful,
-// but WITHOUT ANY WARRANTY; without even the implied warranty of
-// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
-// GNU General Public License for more details.
-//
-// You should have received a copy of the GNU General Public License
-// along with this program. If not, see <https://www.gnu.org/licenses/>.
```
Comment (serhii-temchenko) [inline pallets/liquidity/src/weights.rs]: I checked substrate weights auto generated files and they have copyrights in it as well... this is why I added it here initially. Do you think we should remove copyrights from weights? if so then leave as it is, if not - please add it back.

File: pallets/ocex/src/weights.rs [hunk 2]
```diff
@@ -1,226 +1,340 @@
...
-// the Free Software Foundation, either version 3 of the License, or
-// (at your option) any later version.
-//
-// This program is distributed in the hope that it will be useful,
-// but WITHOUT ANY WARRANTY; without even the implied warranty of
-// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
-// GNU General Public License for more details.
-//
-// You should have received a copy of the GNU General Public License
-// along with this program. If not, see <https://www.gnu.org/licenses/>.
```
Comment (serhii-temchenko) [inline pallets/ocex/src/weights.rs]: Same

File: pallets/rewards/src/weights.rs [hunk 3]
```diff
@@ -1,81 +1,93 @@
...
-// the Free Software Foundation, either version 3 of the License, or
-// (at your option) any later version.
-//
-// This program is distributed in the hope that it will be useful,
-// but WITHOUT ANY WARRANTY; without even the implied warranty of
-// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
-// GNU General Public License for more details.
-//
-// You should have received a copy of the GNU General Public License
-// along with this program. If not, see <https://www.gnu.org/licenses/>.
```
Comment (serhii-temchenko) [inline pallets/rewards/src/weights.rs]: Same

File: pallets/thea/src/benchmarking.rs [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+	verify {
+		assert!(<IncomingNonce::<T>>::get(network) == nonce);
+	}
+}
+//
+//#[cfg(test)]
+//use frame_benchmarking::impl_benchmark_test_suite;
+//
+//#[cfg(test)]
+//impl_benchmark_test_suite!(Thea, crate::mock::new_test_ext(), crate::mock::Test);
```
Comment (serhii-temchenko) [inline pallets/thea/src/benchmarking.rs]: Please remove

File: pallets/thea/src/weights.rs [hunk 5]
```diff
@@ -0,0 +1,99 @@
+
```
Comment (serhii-temchenko) [inline pallets/thea/src/weights.rs:18]: Same about copyrights

**Review by serhii-temchenko — CHANGES_REQUESTED** [review]

File: pallets/liquidity/src/weights.rs (hunk 1 above)
Comment (Gauthamastro) [reply pallets/liquidity/src/weights.rs]: Added it back

**Pull request merged**